js/ui/date_box/ui.date_box.mask.js

```diff
@@ -356,15 +375,21 @@ const DateBoxMask = DateBoxBase.inherit({
         eventsEngine.on(this._input(), addNamespace('dxclick', MASK_EVENT_NAMESPACE), this._maskClickHandler.bind(this));
         eventsEngine.on(this._input(), addNamespace('paste', MASK_EVENT_NAMESPACE), this._maskPasteHandler.bind(this));
         eventsEngine.on(this._input(), addNamespace('drop', MASK_EVENT_NAMESPACE), () => {
-            this._renderDisplayText(this._getDisplayedText(this._maskValue));
-            this._selectNextPart();
+            this._renderSelectedPart();
         });
 
+        eventsEngine.on(this._input(), 'compositionend', this._maskCompositionEndHandler.bind(this));
```

Comment:
Where we unsubscribe this event handler?

```suggestion
        eventsEngine.on(this._input(), addNamespace('compositionend', MASK_EVENT_NAMESPACE), this._maskCompositionEndHandler.bind(this));
```